src/ui/ConsoleView.js

```diff
@@ -21,8 +21,7 @@ type StateType = {
   path: string,
   body: ?string,
   json: ?{},
-  loading: boolean,
-  loaded: boolean,
+  network: null | 'loading' | 'loaded',
```

Comment:
no-loaded instead null? null sounds just less readable. If you really want nullable state it could just be network ?: 'loading' | 'loaded'

Reply:
I'll change it to "unsent" then.

Nullable value and undefined key is different in Flow.

`{ x: null }` is valid value for `{ x: ?any }` and `{ x?: any }`. However, `{}` is only valid value for `{ x?: any }`, because `{ x: ?any }` requires the key `x` to show up. If I use `network?: string`, that means the `network` key could be missing.

Reply:
Interesting - I thought you want to allow unassigned state though.